subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/NotificationServiceTests.kt

```diff
@@ -558,4 +563,61 @@ class NotificationServiceTests {
                 }
         }
     }
+
+    @Test
+    fun `callSuscriber should ask mqttNotifier if the brokerUrl startWith mqtt`() = runTest {
+        val subscription = gimmeRawSubscription().copy(
```

Comment:
by `brokerUrl` you mean the uri in the endpoint in the notification ?

Reply:
Here we are in the NotificationService so it is not yet an mqtt brokerUrl. I can rename it endpoint uri since it is the actual name of the field.

Reply:
`endpoint URI` is good